script/script.js

```diff
@@ -12,4 +12,202 @@ for (let i = 0; i < links.length; i += 1) {
+              map lapora verita.
+              `,
+    LiveLink: '',
+    SourceLink: '',
+    image: './img/Pic.png',
+    privateClasses: {
+      seeProjectBtn: 'see-projec-btn-1',
+      modalContainer: 'modal-container1',
+      modalCloseBtn: 'modal-close-btn1',
+    },
+  },
+  {
+    title: 'Professional Art Printing Data',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry´s standard',
+    technologies: ['html', 'bootstrap', 'css'],
+    details:
+              `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's 
+              standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s 
+              with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's 
+              standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax 
+              map lapora verita.
+              `,
+    LiveLink: '',
+    SourceLink: '',
+    image: './img/Pic.png',
+    privateClasses: {
+      seeProjectBtn: 'see-projec-btn-1',
+      modalContainer: 'modal-container1',
+      modalCloseBtn: 'modal-close-btn1',
+    },
+  },
+  {
+    title: 'Professional Art Printing Data',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry´s standard',
+    technologies: ['html', 'bootstrap', 'css'],
+    details:
+              `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's 
+              standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s 
+              with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's 
+              standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax 
+              map lapora verita.
+              `,
+    LiveLink: '',
+    SourceLink: '',
+    image: './img/Pic.png',
+    privateClasses: {
+      seeProjectBtn: 'see-projec-btn-1',
+      modalContainer: 'modal-container1',
+      modalCloseBtn: 'modal-close-btn1',
+    },
+  },
+];
+document.addEventListener('DOMContentLoaded', () => {
+  const projectsContainer = document.querySelector('.posts');
+  let projectContent;
+  let projectLanguages;
+  let projectCardContainer;
+  let modalContent;
+  let modalContainer;
+  projects.forEach((project) => {
+    projectLanguages = project.technologies.map(
+      (lang) => `<li>${lang}</li>`,
+    );
+    projectContent = `
+                <h2>${project.title}</h2>
+                <p>${project.description}</p>
+                <ul class="programming-languages-1">
```

Comment:
- [ ] Please style the color of this `h2` text according to the design specification. The appropriate color should be a dark bluish color #172b4d 👍